hikari/impl/bot.py

```diff
@@ -167,6 +167,11 @@ class GatewayBot(traits.GatewayBotAware):
         Note that this only applies to the REST API component that communicates
         with Discord, and will not affect sharding or third party HTTP endpoints
         that may be in use.
+    max_retries : typing.Optional[builtins.int]
+        Maximum number of times a request will be retried if
+        it fails with a `5xx` status. Defaults to 3 if set to `builtins.None`.
+
+        If you provide a value above 5, it will default to 5.
```

Comment:
If we're gonna have a cap for this it should really error if they try to go over the cap to be explicit instead of implicit

Reply:
Instead of throwing an error, I think that it should rather log it onto the console like "A value higher than 5 has been provided for max_retries, going for 5 retries"